Tasks/PowerShellV2/powershell.ps1

```diff
@@ -34,7 +36,8 @@ try {
         }
 
         $input_arguments = Get-VstsInput -Name 'arguments'
-    } else {
+    }
```

Comment:
Nit: could you please make formatting for curly braces consistent as it was before for this file?